### PR TITLE
Bump to @polkadot/api ^7

### PR DESCRIPTION
**Pull request by fishmonger45**
- Update `docker-compose.yml`
- Bump to specVersion `v0.2.0`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -42,4 +57,4 @@ services:
     command:
       - --name=app
       - --playground
-
+      - --indexer=http://subquery-node:3000
```
Comment (ianhe8x) [inline docker-compose.yml]: The usage of `--indexer` has been deprecated.

File: src/types/models/BlockTs.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 // Auto-generated , DO NOT EDIT
-import {Entity} from "@subql/types";
+import {Entity, FunctionPropertyNames} from "@subql/types";
```
Comment (ianhe8x) [inline src/types/models/BlockTs.ts]: I thought we don't submit auto-generated files.

**Review by ianhe8x — APPROVED** [review]
You can merge it after you fix the other places.

File: docker-compose.yml (hunk 1 above)
Comment (fishmonger45) [reply docker-compose.yml]: we should mark it as such on yargs as well

**Pull request merged**